pyne/mcnp.py

```diff
@@ -1415,19 +1417,24 @@ def mat_from_inp_line(filename, mat_line, densities='None'):
     # create dictionaries nucvec and table_ids
     nucvec = {}
     table_ids = {}
+    table_ids['default'] = {}
     for i in range(1, len(data_string.split())):
         if i & 1 == 1:
+            # Eliminates default library indicators
```

Comment:
I just realized that this won't work!!!  The logic of this `i & 1 == 1` threw me off

Reply:
Still there may be a way to get over it! I am on it.